Xamarin.PropertyEditing.Mac/Controls/EnumEditorControl.cs

```diff
@@ -9,11 +9,12 @@
 
 namespace Xamarin.PropertyEditing.Mac
 {
-	internal class EnumEditorControl : PropertyEditorControl
+	internal class EnumEditorControl<T> : PropertyEditorControl
```

Comment:
@ermau I think this change to make the control generic, as per @alanmcgovern 's suggestion is a more elegant and cleaner fix than jumping through the Activator etc hoops.